api/v1alpha1/eip_types.go

```diff
@@ -25,6 +25,7 @@ type EipSpec struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Pattern=`^([0-9]{1,3}\.){3}[0-9]{1,3}(\/([0-9]|[1-2][0-9]|3[0-2]))?$`
 	Address       string `json:"address,omitempty"`
+	LBType        string `json:"lbType,omitempty"`
```

Comment:
How about change `LBType` to `Protocol` like metalLB

Reply:
I don't think it matters, maybe it's better to use different fields